addons/survey/static/src/js/survey_result.js

```diff
@@ -3,6 +3,7 @@ odoo.define('survey.result', function (require) {
 
```

Comment:
@angv-odoo  Oh I forgot, for the commit message, we usually use Task- instead of Ticket- ...
So here `task-3092439`

Reply:
I see it is done on the PR message but you can also do it in the Commit message :)